.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "fastR-examples"]
```

Comment:
The submodule repository was archived by the owner on May 22, 2024:
https://github.com/graalvm/fastr-examples/tree/423444367e03901f04a8f485a31a6c3002f6a76f